src/autocomplete/CommandProvider.js

```diff
@@ -83,4 +83,10 @@ export default class CommandProvider extends AutocompleteProvider {
 
         return instance;
     }
+
+    renderCompletions(completions: [React.Component]): ?React.Component {
+        return React.cloneElement(super.renderCompletions(completions), {
+            className: 'mx_Autocomplete_Completion_container_block',
+        });
+    }
```

Comment:
Given the superclass function here is taking the array and putting it in a div wrapper, the inheritance here seems a bit redundant, since it's doing this wrapping for the overridden function to clone it, add a class, return that and throw the original away.